pkg/capture/GPPacket.go

```diff
@@ -158,10 +169,20 @@ func (p *GPPacket) Populate(pkt capture.Packet) error {
 	return nil
 }
 
+func isNotCommonPort(port []byte) bool {
+	return port[0] != 0 || // Certainly not interesting
+		port[1] != 53 // Not DNS
+
+	// TODO: Potentially extend to more common ports?
```

Comment:
@els0r Here I'd like your opinion: I was wondering if it might make sense to extend this logic to a (very select) few very well known ports where we can be relatively certain that we don't miss out on some important flows, e.g. HTTP & HTTPS. Rationale: When testing I saw that really quite a lot of HTTP / HTTPS connections were created for the same SIP/DIP pair from various source ports more or less at the same time (probably multiple threads / chunking), so similar to DNS requests this clutters the flow map quite a lot, without any gain in knowledge (because those will get merged upon rotation anyway)... What's your take on that?

Reply:
I’m with you on this. And it’s interesting you come to the same conclusion. For whatever reason, I did a repeated `curl` on the `flows` endpoint when capturing the other day and watched the flow map expand and compress across write out cycles (which also confirms that we/I broke the write out in the new state machine after it was already working properly - probably as part of a refactor or review feedback cycle 🤦 ).

Back to topic: It was visually apparent that the flow detection and keeping of HTTP(S) flows didn’t bring much to the table, due to the fact that new sessions would be opened up on some other source ports. So a lot of time is spent classifying and copying over flows that are essentially cleaned up a cycle later.

If we stick with one of goProbe’s early design decisions to disregard the source port, I would say let’s do what you propose 👍 .

Reply:
OK, then let's start with port 80 / 443, since that's a low hanging fruit that I can just uncomment and we can use it to see if we run into any issues with that. If there's other obvious destination ports that can be added we can do so later (doesn't even have to be as part of the v4 release because it shouldn't (tm) change the final flow map).